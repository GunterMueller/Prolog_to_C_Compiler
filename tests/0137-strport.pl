:- include(library(openstr)).

main :-
	open_input_string('%...\nfoo("a", 123). ', S),
	read(S, X),
	writeq(X), nl,
	open_output_string(S2),
	fwritef(S2, "%d skiddoo\n%f", [23]),
	get_output_string(S2, Y),
	writef("%s", [Y]).

