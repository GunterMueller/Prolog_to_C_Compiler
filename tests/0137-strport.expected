foo([97],123)
23 skiddoo
